SMBPositionAnalytics.md

```diff
@@ -0,0 +1,151 @@
+A request to the carotene endpoint provides data on how a specified job title performs in a given location.
+
+E.g. I'm a recruiter and I'm about to post a Software Engineer job in Atlanta. How many applies have there been to Software Engineer titles in this city in the last day? What is the change from the day before? On average how long does it take to fill a Software Engineer role in this location? This endpoint serves to answer these questions.
+
+Requests consist of:
+
+| Param | Type | Description |
+|-------|------|-------------|
+| `carotene_id` | string | Unique carotene identifier v3.1.
+| `city` | string | Full name of a city respective of the carotene title
+| `state` | string | Two letter abbreviation for the state
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/positionanalytics/carotene \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"carotene_id": "11.0",
+	"city": "Atlanta",
+	"state": "GA",
+      }'
+```
+
+Currently only Carotene v3.1 is supported. See the documentation for [Job Title service](https://github.com/careerbuilder/DataScienceAPIDocumentation/blob/master/JobTitle.md)
+for more information about carotene classification and carotene IDs.
+
+### Job Request
+A request to the job endpoint requires a CareerBuilder Job Identifier also known as a Job DID and serves data pertaining to active jobs(open positions). Relevant data points returned include: number of clicks, number of applies, number of views, and total number of actions.
+
+Requests consist of:
+
+| Param | Type | Description |
+|-------|------|-------------|
+| `job_did` | string | Unique CareerBuilder job identifier.
+
+
+Example cURL request:
+
+```
+curl -X POST \
+  https://api.careerbuilder.com/core/positionanalytics/job \
+  -H 'Accept: application/json;version=1.0' \
+  -H 'Authorization: <BEARER_TOKEN>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+	"job_did": "J2N21P76BM530W9HRQG",
+      }'
+```
+
+## Responses
+The data returned in the response will contain an object or group of objects representing actions taken on a carotene title or job respectively. Each action will have associated `current` and `delta` values. The `current` value will account the total number of times an action was performed over the last day; the `delta` will identify the change in percentage of the `current` value from the day before. Positive values in the `delta` identify an increase day over day and negative values identify a decrease.
+
+ Each response will return a trailing `timestamp` string. This is the date time in which the data was last refreshed. The `timestamp` string is returned in ISO 8601 format with UTC offset(yyyy-MM-dd'T'HH:mm:ss-XXX). Clients can expect that data will be refreshed daily to account for the changes to the `current` and  `delta` values which outline actions performed in the last day.
```

Comment:
Oh are we returning the timestamp? I didn't realize we were. I remember us talking about standardizing it but I guess I didn't realize what that was for. Hmm....I'm not sure we want to be returning this (ponder).